src/civis/_utils.py

```diff
@@ -31,16 +29,6 @@ def maybe_get_random_name(name):
     return name
 
 
-def camel_to_snake(word):
-    # https://gist.github.com/jaytaylor/3660565
-    word = UNDERSCORER1.sub(r"\1_\2", word)
-    return UNDERSCORER2.sub(r"\1_\2", word).lower()
-
-
-def to_camelcase(s):
```

Comment:
The only module that uses `to_camelcase` is `service_client.py`, so I'm moving this function there.